### ISSUE TITLE
[ERROR] Failed to load the book due to an unexpected error.  + Construct's Armory

### ISSUE THREAD

**Issue by SolSavvy** [SlimeKnights/TinkersConstruct#3616]
**Issue description: The book, Materials and You, just suddenly stopped loading; mid game. This is also the case with Construct's Armory version of the book of the same name.  The error is shown as NullPointerException .**

**If crashed, link to crash report (use a site such as pastebin): No crash as of yet.**

**Versions:**
* Minecraft: 1.12.2
* Forge: 14.23.4.2750
* Mantle: 1.3.2.24
* Tinkers Construct: 1.12.2-2.10.1.87
* Construct's Armory: 1.12.2-1.0.2

**Can it be reproduced with just Tinkers Construct, or are other mods required to reproduce the bug?**

Materials and You
![2018-08-16_10 39 04](https://user-images.githubusercontent.com/27328200/44216603-4c501e80-a143-11e8-904b-179cee778e22.png)

Materials and You [Construct's Armory]
![2018-08-16_10 42 11](https://user-images.githubusercontent.com/27328200/44216749-a9e46b00-a143-11e8-8223-7ec5ea9d2c3c.png)

**Comment by rylm** [comment]
I have the same issue.

Minecraft: 1.12.2
Forge: 14.23.4.2747
Tinkers Construct: 1.12.2-2.10.1.87
Mantle: 1.12-1.3.2.24

**Comment by fuj1n** [comment]
SlimeKnights/Mantle@242ffd9

**Comment by KnightMiner** [comment]
Yeah, somehow it seems a recent launcher update broke languages so they are all missing. You should find your language menu is empty. The should be caught now as of SlimeKnights/Mantle@242ffd9, so next mantle release will at least make it have a nice error of not work.

**Comment by rylm** [comment]
@KnightMiner is there a way to add this forge fix to Twich/Curse setup right away?

**Comment by KnightMiner** [comment]
If it's not there, you will just have to wait or use the vanilla launcher
